platform/linux-generic/odp_schedule_sp.c

```diff
@@ -819,6 +819,11 @@ static void schedule_order_unlock(unsigned lock_index)
 	(void)lock_index;
 }
 
+static void schedule_order_unlock_lock(unsigned lock_index)
+{
+	(void)lock_index;
```

Comment:
I didn't realize the schedule_sp didn't implement ordered locks. @psavol is this a "to do" item?

Reply:
@psavol The SP scheduler only exists to provide strict priority scheduling? The scalable scheduler does that as well (and doesn't the default scheduler also do basic SP scheduling?). Are there any specific characteristics of the SP scheduler (beyond the basic SP scheduling) that are of interest? In benchmarking, the SP scheduler seemed to be the slowest by far.

Reply:
It implements ordered lock API by handling all queues as atomic.

It demonstrates a very simple implementation of the API, and thus is an easy starting for new scheduler development.